pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -610,13 +611,16 @@ func (r *Reconciler) updateShootStatusOperationStart(
 		LastUpdateTime: now,
 	}
 
-	var mustRemoveOperationAnnotation bool
+	var (
+		operation                     = shoot.Annotations[v1beta1constants.GardenerOperation]
+		mustRemoveOperationAnnotation bool
+	)
```

Comment:
```suggestion
	)

```

newline for readability